AATAMS/AATAMS_sattag_dm/lftp_sync.sh

```diff
@@ -23,10 +23,15 @@ sync_files() {
 # main
 main() {
     mkdir -p $ZIPPED_DIR
-    local tmp_lftp_output_file=`mktemp`
-    sync_files $tmp_lftp_output_file
 
-    mv $tmp_lftp_output_file $INCOMING_DIR/AATAMS/AATAMS_SATTAG_DM/aatams_sattag_dm_lftp.`date +%Y%m%d-%H%M%S`.log
+    # look for empty dir
+    if [ "$(ls -A $ERROR_DIR/AATAMS_SATTAG_DM)" ]; then
+         echo "Unable to run lftp as $ERROR_DIR/AATAMS_SATTAG_DM is not empty"
```

Comment:
Have no specific knowledge of this specific script, but if the script generates any output, Cron should email it to the MAILTO address. @lbesnard is this your expectation?

https://github.com/aodn/data-services/blob/master/cron.d/AATAMS_SATTAG_DM